articles/partner-twilio-cloud-services-dotnet-phone-call-web-role.md

```diff
@@ -90,19 +93,19 @@ The following code, which is called when the user completes the form, creates th
 
                 // Use your account SID and authentication token instead of
                 // the placeholders shown here.
-                string accountSID = "ACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN";
-                string authToken =  "NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN";
+                var accountSID = "ACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN";
+                var authToken =  "NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN";
 
                 // Instantiate an instance of the Twilio client.
-                TwilioRestClient client;
-                client = new TwilioRestClient(accountSID, authToken);
+                var client = new TwilioRestClient(accountSID, authToken);
 
                 // Retrieve the account, used later to retrieve the
-                Twilio.Account account = client.GetAccount();
-                string APIversuion = client.ApiVersion;
-                string TwilioBaseURL = client.BaseUrl;
+                var account = client.GetAccount();
+                var APIversion = client.ApiVersion;
```

Comment:
From the naming convention, this should probably be in camel case like so `apiVersion`